packages/@aws-cdk/aws-apigateway/lib/authorizer.ts

```diff
@@ -1,9 +1,26 @@
+import { Construct } from '@aws-cdk/core';
+import { AuthorizerBase, TokenAuthorizer, TokenAuthorizerProps } from './authorizers';
+
 /**
  * Represents an API Gateway authorizer.
  */
 export interface IAuthorizer {
   /**
    * The authorizer ID.
+   * @attribute
    */
   readonly authorizerId: string;
+}
+
+/**
+ * Base class for all custom authorizers
+ */
+export class Authorizer {
+
+  /**
+   * Return a new token authorizer with the specified properties.
+   */
+  public static token(scope: Construct, id: string, props: TokenAuthorizerProps): AuthorizerBase {
```

Comment:
We normally use the "Props" postfix only for constructor keyword arguments. Otherwise, it's `TokenAuthorizerOptions`.

Reply:
That's true when defining an instance method on a construct, but this is a static method. In this case, `TokenAuthorizerProps` === `TokenAuthorizerOptions`.

Would you still create an empty `TokenAuthorizerProps` that only extends `TokenAuthorizerOptions`?